ferrocene/doc/user-manual/src/rustfmt/rustfmt-cli.rst

```diff
@@ -0,0 +1,86 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Command-Line Interface
+======================
+
+.. cli:program:: rustfmt
+
+   .. cli:option:: --check
+
+      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
+
+   .. cli:option:: --emit [files|stdout|coverage|checkstyle|json]
+
+      What data to emit and how
+
+   .. cli:option:: --backup
+
+      Backup any modified files.
+
+   .. cli:option:: --config-path
+
+      Recursively searches the given path for the rustfmt.toml config file. If not found reverts to the input file path
+
+   .. cli:option:: --edition [2015|2018|2021]
+
+      Rust edition to use
```

Comment:
What changes when changing the edition? We don't need to list all changes, but at least mention the general shape of what someone could expect.